### DISCUSSION TITLE
Cleaning up the core teams

### DISCUSSION THREAD

**Discussion by axel-lauer** [ESMValGroup/Community#151] [General]
The ESMValTool proto-Steering Group proposes to start enforcing our no-activity policy, i.e. cleaning up the core teams and removing members who have been inactive for more than 2 years. In particular, we propose the following:

* Enforcing our no-activity policy, members of the [Scienctific Lead Development Team](https://github.com/orgs/ESMValGroup/teams/scientific-lead-development-team), the [Technical Lead Development Team](https://github.com/orgs/ESMValGroup/teams/technical-lead-development-team), and the [User Engagement Team](https://github.com/orgs/ESMValGroup/teams/userengagementteam) who have **not been active for more than 2 years** will be removed from these teams. All members removed from these teams will remain in the [ESMValTool Development Team](https://github.com/orgs/ESMValGroup/teams/esmvaltool-developmentteam).
* The GitHub team [ESMValTool-CoreTeam](https://github.com/orgs/ESMValGroup/teams/esmvaltool-coreteam) became obsolete will be removed.
* The startup message listing the [ESMValTool-CoreTeam](https://github.com/orgs/ESMValGroup/teams/esmvaltool-coreteam) and contacts in [_main.py](https://github.com/ESMValGroup/ESMValCore/blob/main/esmvalcore/_main.py) will be replaced by the three mailing lists for the [Scienctific Lead Development Team](https://github.com/orgs/ESMValGroup/teams/scientific-lead-development-team), the [Technical Lead Development Team](https://github.com/orgs/ESMValGroup/teams/technical-lead-development-team), and the [User Engagement Team](https://github.com/orgs/ESMValGroup/teams/userengagementteam).

All of this would be implemented for the next release, which is currently scheduled for **April 2024**.

Please give us feedback on this idea and share your thoughts or concerns on this proposed cleanup.

@ESMValGroup/esmvaltool-developmentteam
@ESMValGroup/esmvaltool-coreteam 
@ESMValGroup/scientific-lead-development-team 
@ESMValGroup/technical-lead-development-team 
@ESMValGroup/userengagementteam

**Comment by valeriupredoi** [comment]
my feedback: :broom: :+1: x3 - well done! Note that the April release will be a bit delayed, possibly May :gb:

**Comment by bouweandela** [comment]
Sounds good to me! We should be careful that we do recognize all kinds of activity, not just activity on GitHub, but also e.g. help with finding funding.

**Comment by bouweandela** [comment]
Would it make sense to list the members of the respective teams on our website or on readthedocs? Or are the GitHub teams sufficient? Note that we currently have [this page](https://docs.esmvaltool.org/en/latest/introduction.html#core-development-team), which would need to be updated. @ehogan recently suggested that we could write downs somewhere what people's specific skills or interests are, so it would be more clear to everyone who to contact for what.

**Reply by bouweandela** [reply]
Here are some examples of how other projects list teams/roles and people:
- [astropy](https://www.astropy.org/team.html#role-responsibilities)
- [sunpy](https://sunpy.org/about/roles)
- [rucio](https://rucio.cern.ch/team.html)
- The Turing Way [roles](https://the-turing-way.netlify.app/afterword/contributors-record#project-delivery-team-members), [collaborators](https://the-turing-way.netlify.app/afterword/collaborators), [contributors](https://github.com/the-turing-way/the-turing-way?tab=readme-ov-file#contributors)

**Comment by axel-lauer** [comment]
I propose to keep this discussion open until the upcoming workshop in May 2024 (see https://github.com/ESMValGroup/Community/discussions/141) and then start enacting (i.e. contacting non-active members) what we decide then.

**Comment by headmetal** [comment]
Perhaps for long-term (but currently inactive) contributors, a new group called something like "Founding members" could be established?
